src/client/components/LanguageSelector.tsx

```diff
@@ -39,6 +39,11 @@ export const LanguageSelector: React.FC = React.memo(() => {
             key: 'ru',
             itemIcon: <CountryFlag code="ru" />,
           },
+          {
+            label: 'Português',
+            key: 'pt',
+            itemIcon: <CountryFlag code="pt" />,
+          },
```

Comment:
can you move above `ru`, because the letter `p` is before to `r`

Reply:
All done, feel free to confirm